ClangSharpPInvokeGenerator/Program.cs

```diff
@@ -151,12 +153,12 @@ private static void AddAdditionalOption(RootCommand rootCommand)
         {
             var argument = new Argument();
             argument.ArgumentType = typeof(string);
-            argument.Arity = ArgumentArity.ZeroOrMore;
+            argument.Arity = ArgumentArity.OneOrMore;
```

Comment:
`ZeroOrMore` was actually allowing you to specify `--additional` with no argument